ccm_web/client/src/components/BulkSectionCreateValidators.ts

```diff
@@ -1,55 +1,24 @@
 import { InvalidationType } from '../models/models'
+import { DuplicateIdentifierInRowsValidator, RowInvalidation, StringRowsSchemaValidator } from '../utils/rowValidation'
 import { sectionNameSchema, validateString } from '../utils/validation'
```

Comment:
There is an linting error saying `validateString` not being used.

Reply:
Fixed. https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/270/commits/6a05e07b312bfa996844678ccbe0fcfca46c4441